google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerErrorInterceptor.java

```diff
@@ -69,6 +71,12 @@ public void start(Listener<RespT> responseListener, Metadata headers) {
               @Override
               public void onClose(Status status, Metadata trailers) {
                 try {
+                  // Translate INTERNAL errors that should be retried to a retryable error code.
+                  if (status.getCode() == Code.INTERNAL
```

Comment:
I can see the same check in `isRetryableInternalError` as well. Can we remove it from here?

Reply:
yeah, good point. I initially had this set up in a way where only the error message was checked in the RetryableInternalError class, and forgot to remove this after that refactor. Removed now.